.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,12 +1,12 @@
 ### Description of the Changes
 
-Please add a detailed description of the change, weather it's an enhancement or a bugfix.
+Please add a detailed description of the change, whether it's an enhancement or a bugfix.
 If the PR is related to an open issue please link to it.
 
-### CheckLists
+**Issue:**
```

Comment:
why change this file

Reply:
i just formalizing the existing consensus